docs/installation-and-operations/operation/restoring/README.md

```diff
@@ -241,3 +260,41 @@ You may need to create the `files` directory if it doesn't exist yet.
 
 Start the container as described in the [installation section](../../installation/docker/#one-container-per-process-recommended)
 mounting `/var/lib/openproject/pgdata` (and `/var/lib/openproject/assets/` for attachments).
+
+## Changing the database schema from cloud to on-premises
+
+At the restore from the Cloud to the Local on-premises instance you should beware of the following steps:
+
+1. Stop OpenProject on-premises  but keep the database up and running.
+2. Connect to your PSQL database.
+
+2. The `public` schema in the on-premises database needs to be dropped if it exists.
+
+```psql
+openproject=# DROP SCHEMA public CASCADE;
+DROP SCHEMA
+```
+
+3. The cloud's database schema name can be listed with the following command.
+
+```psql
+openproject=# \dn
+(...)
+```
+
+3. The schema with "name of a long number" needs to be renamed after import to `public`.
+
+```psql
+openproject=# ALTER SCHEMA "123456789_1234567_1234567a_123b_12c3_1234_c2a1a123c123" RENAME TO "public";
+ALTER SCHEMA
+```
+
+4. OPTIONAL: The `protocol` in the `settings` table could be changed from `https`to `http`. (This is only recommended if you do not like to use HTTPS)
```

Comment:
These steps are no longer valid, as the `protocol` setting has been removed in 12.2

The setting is called `https` and gets values 'true' and 'false'.

Reply:
i removed the OPTIONAL part in order have less information because it is off topic